Jenkinsfile

```diff
@@ -37,14 +37,28 @@ pipeline {
       environment {
         CF_APP_NAME="portaljs${env.CF_SPACE == 'production' ? '' : '-' + env.CF_SPACE}"
         CTF_CPA_ACCESS_TOKEN=credentials("portaljs.${env.CF_SPACE}.contentful.cpa")
+        LOGSTASH_CONNECTION=credentials("elk_connection_string")
```

Comment:
This isn't really a credential as it's just a username and hostname, plus it's the same for all jobs on our stack.

Let's therefore move it to a [global Jenkins env var](https://jenkins.eanadev.org/configure) (as with `CF_API` and `CF_ORG`), named `ELK_LOGSTASH_SSH_DESTINATION`.